src/models/backup.ts

```diff
@@ -157,6 +166,11 @@ class Drive {
 
   private async getFolder() {
     const token = await this.getToken();
+    if (!token) {
+      return new Promise((resolve: (value: false) => void) => {
```

Comment:
do you mean `value: boolean` ?

Reply:
Since it is only supposed to return false is it ok to just leave it like this or is it better to use Boolean?

Reply:
`false` is not a type, `boolean` should be used here.